### PR TITLE
Upgrade requirements

### PR DESCRIPTION
**Pull request by flatnine**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -12,11 +12,21 @@ setenv = AWS_ACCESS_KEY_ID = fakekey
...
 
+[testenv:py36]
+install_command = pip install --no-deps {opts} {packages}
+deps = -r{toxinidir}/requirements-py36.txt
+commands = pytest {posargs} tests/
+# Make local aws credentials get ignored
+setenv = AWS_ACCESS_KEY_ID = fakekey
+         AWS_SECRET_ACCESS_KEY = fakesecret
+         AWS_DEFAULT_REGION = eu-west-1
+
```
Comment (adamchainz) [inline tox.ini]: you only need to specify the keys that are different to the default `testenv` here, i.e. only `deps`

**Comment by flatnine** [comment]
@adamchainz  So as of this month `responses` uses a library called `biscuits` for python versions > 3.4.  Earlier versions use the `cookies` library.   The method I used seemed to be the simplest solution though I'm not sure if it's best practice.  

I'll tidy up the tox.ini file with your suggestions above.  

Thanks for taking a look.

File: tox.ini (hunk 1 above)
Comment (flatnine) [reply tox.ini]: Removed.

**Comment by adamchainz** [comment]
Oh yeah, because of this and other reasons I moved off `responses`, e.g. in ec2-metadata: https://github.com/adamchainz/ec2-metadata/pull/97

Told them here https://github.com/getsentry/responses/pull/221#issuecomment-431683378

Also they're reverting the use of `biscuits`: https://github.com/getsentry/responses/pull/224

**Pull request merged**